### DISCUSSION TITLE
2.5 面向对象编程

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#110] [Announcements]
# 2.5 面向对象编程

SICP Python 中文翻译

https://composingprograms.netlify.app/2/5

**Comment by MeJinx** [comment]
这章写的好高明啊，确实比国内的教材要深入透彻。

**Comment by HelloHJ789** [comment]
这本书中的每一句话都特别精炼抽象，每一句话都值得细细品味

**Comment by haoaaa-111** [comment]
真不知道要是零基础咋开这个 好多名词要是没有经验真和看天书一样

**Reply by AB203-eng** [reply]
CS61A本身就不是零基础该学的课程吧

**Comment by sisyphecamus** [comment]
好！
